components/Ripple/BUILD

```diff
@@ -19,7 +19,10 @@ load(
     "mdc_objc_library",
     "mdc_public_objc_library",
     "mdc_unit_test_suite",
+    "mdc_snapshot_objc_library",
```

Comment:
Can these BUILD file changes be moved to a separate PR or are they somehow required for the Podspec changes?

Reply:
In a previous iteration of this PR I had added a Swift source dummy to `components/Ripple/src/` so I had to add the snapshot tests to the BUILD otherwise CI wouldn't pass. I can get rid of this now I guess...